Makefile

```diff
@@ -17,20 +17,35 @@ all: build run
 run:
 	./${OUT}
 
-build: go/generate go/format
+run/parallel:
+	cat config/config.json | jq -r '.peers | .[].name' | parallel ./build/rfs -n {} | jq
```

Comment:
Instead of piping cat, you can simply run `jq -r '.peers | .[].name' config/config.json`.

Reply:
fair point :)